packages/bootstrap/scss/action-sheet/_variables.scss

```diff
@@ -0,0 +1,29 @@
+// Action-sheet
+$action-sheet-width: 372px !default;
+$action-sheet-max-height: 50vh !default;
+$action-sheet-box-shadow: 0 8px 10px -5px rgba(0, 0, 0, .2), 0 16px 24px 2px rgba(0, 0, 0, .14), 0 6px 30px 5px rgba(0, 0, 0, .12) !default;
```

Comment:
I understand that this is the shadow from the design, however, the element in the same design has a different shadow. Also, do we need another type of shadow in our code? Can you ask a designer if it is okay to use the Popup shadow in this case, or which of the two shadows in the design we should use? @joneff do you have input on this?

![image](https://user-images.githubusercontent.com/17339722/110119905-0de3e300-7dc5-11eb-9835-a0d976c59b93.png)

Reply:
We want the Action Sheet to have consistent shadow across all themes,  the multiple shadows add the spatial effect to highlights the importance of the active Action Sheet, making the component appear "lifted" and "floating" above all other elements under/behind it.